src/Explorer/Tables/TableDataClient.ts

```diff
@@ -172,8 +172,12 @@ export class CassandraAPIDataClient extends TableDataClient {
           deferred.resolve(entity);
         },
         (error) => {
-          handleError(error, "AddRowCassandra", `Error while adding new row to table ${collection.id()}`);
-          deferred.reject(error);
+          handleError(
+            error.responseJSON?.message ?? JSON.stringify(error),
+            "AddRowCassandra",
+            `Error while adding new row to table ${collection.id()}`,
+          );
+          deferred.reject(error.responseJSON?.message ?? JSON.stringify(error));
```

Comment:
Should these stay as deferred.reject(error) ?

Reply:
This returns the error in the side pane as well.
![image](https://github.com/Azure/cosmos-explorer/assets/144163838/fc16df46-1791-4db3-80d9-4096f0890c8b)

If it's just error, no error will appear in the side pane since error is always undefined.